domaintools_phisheye/help.md

```diff
@@ -0,0 +1,118 @@
+# Description
+
+[DomainTools PhishEye](https://www.domaintools.com/resources/user-guides/phisheye) is a potent tool in your battle against these nefarious domains and the criminals who operate them. Leveraging DomainTools’ unique visibility into over 315 million current domains. PhishEye finds domains that mimic your properties, whether by typo or by combining your brand with other terms—terms like “account,” “login,” “online,” or countless others.
```

Comment:
```suggestion
[DomainTools PhishEye](https://www.domaintools.com/resources/user-guides/phisheye) is a potent tool in your battle against nefarious domains and the criminals who operate them. Leveraging DomainTools’ unique visibility into over 315 million current domains, PhishEye finds domains that mimic your properties, whether by typo or by combining your brand with other terms—terms like “account,” “login,” “online,” or countless others.
```

Reply:
This should be only in help.md only or in plugin.spec also?

Reply:
help.md - Plugin spec has the shorter description. Applying Github suggestions here.